spark/v3.3/spark/src/main/java/org/apache/iceberg/spark/ChangelogIterator.java

```diff
@@ -89,15 +91,29 @@ private ChangelogIterator(
    * @param rowType the schema of the rows
    * @param identifierFields the names of the identifier columns, which determine if rows are the
    *     same
+   * @param netChanges whether to compute net changes across all snapshots
    * @return a new {@link ChangelogIterator} instance concatenated with the null-removal iterator
    */
   public static Iterator<Row> create(
```

Comment:
This is a public API, I'm not sure we should change the signature? Can we just add a new one and default to false?